x-pack/plugins/profiling/public/components/flamegraph/flamegraph_tooltip.tsx

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { TooltipContainer } from '@elastic/charts';
+import {
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiHorizontalRule,
+  EuiIcon,
+  EuiPanel,
+  EuiText,
+  useEuiTheme,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { isNumber } from 'lodash';
+import React from 'react';
+import { asCost } from '../../utils/formatters/as_cost';
+import { asPercentage } from '../../utils/formatters/as_percentage';
+import { asWeight } from '../../utils/formatters/as_weight';
+import { calculateImpactEstimates } from '../flame_graphs_view/calculate_impact_estimates';
+import { TooltipRow } from './tooltip_row';
+
+interface Props {
+  isRoot: boolean;
+  label: string;
+  countInclusive: number;
+  countExclusive: number;
+  totalSamples: number;
+  totalSeconds: number;
+  baselineScaleFactor?: number;
+  comparisonScaleFactor?: number;
+  comparisonCountInclusive?: number;
+  comparisonCountExclusive?: number;
+  comparisonTotalSamples?: number;
+  comparisonTotalSeconds?: number;
+  onShowMoreClick?: () => void;
+}
+
+export function FlameGraphTooltip({
+  isRoot,
+  label,
+  countInclusive,
+  countExclusive,
+  totalSamples,
+  totalSeconds,
+  baselineScaleFactor,
+  comparisonScaleFactor,
+  comparisonCountInclusive,
+  comparisonCountExclusive,
+  comparisonTotalSamples,
+  comparisonTotalSeconds,
+  onShowMoreClick,
+}: Props) {
+  const theme = useEuiTheme();
+
+  const impactEstimates = calculateImpactEstimates({
+    countExclusive,
+    countInclusive,
+    totalSamples,
+    totalSeconds,
+  });
+
+  const comparisonImpactEstimates =
+    isNumber(comparisonCountExclusive) &&
+    isNumber(comparisonCountInclusive) &&
+    isNumber(comparisonTotalSamples) &&
+    isNumber(comparisonTotalSeconds)
+      ? calculateImpactEstimates({
+          countExclusive: comparisonCountExclusive,
+          countInclusive: comparisonCountInclusive,
+          totalSamples: comparisonTotalSamples,
+          totalSeconds: comparisonTotalSeconds,
+        })
+      : undefined;
+
+  return (
+    <TooltipContainer className="Caue-tooltip">
```

Comment:
Is the class name a find-replace error? ;)

Reply:
😅